mods/d2k/rules/arrakis.yaml

```diff
@@ -35,9 +35,7 @@ SANDWORM:
 			Dune: 100
 			Spice: 100
 	TargetableUnit:
-		TargetTypes: Underground
-	RevealsShroud:
-		Range: 32c0
+		TargetTypes: Ground
```

Comment:
IIRC the worms in the original could only be attacked when they were above ground.  We can adjust this in a future pr.